server/views/forms/summary-offence-analysis.njk

```diff
@@ -0,0 +1,30 @@
+{% extends "./layout.njk" %}
```

Comment:
Does this need removing now `Offence analysis` uses the generic summary template?

Reply:
Actually, i can delete the whole file as we now use summary-new.njk to render the summary page instead.